samples/java-customer-registry-quickstart/README.md

```diff
@@ -49,6 +49,9 @@ To start the application locally, the `exec-maven-plugin` is used. Use the follo
 mvn compile exec:java
 ```
 
+> On Linux, the application will bind to localhost which will not be accessible from the docker containers depending on the
+> docker networking model. Use `HOST=0.0.0.0 mvn compile exec:java`, instead to make the user application bind to all interfaces.
```

Comment:
was this sufficient in your case? on my machine I also have to open up some firewall rules, but perhaps that's uncommon?

Reply:
Yes, seems it was enough. What kind of firewall do you use or rules do you set?

Reply:
> What kind of firewall do you use

I don't really know, the defaults from NixOS :). I can dump the rules if you want?

I ended up just doing `iptables -I INPUT -i the_interface -j ACCEPT` for the interface docker-compose was using.

Reply:
Ok, thanks. Maybe NixOS is just more locked down by default...